### PR TITLE
fixes shadow and form footer for ksvc form

### PR DESCRIPTION
**Pull request by invincibleJai**
**Fixes**: 
https://issues.redhat.com/browse/ODC-5589

**Analysis / Root cause**: 
Form footer and shadow effect not proper for the form created for knative service

**Solution Description**: 
used `FlexForm` and `FormBody` to wrap content and removed the unwanted wrapping

**Screen shots / Gifs for design review**: 

image

Mar-03-2021 16-18-08

**Test setup:**

- Install Serverless Operator
- Create a workload D/DC from container image or import flows
- Can see the option to "Create Knative service" once the workload is in ready state
- View the form footer

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by invincibleJai** [comment]
Holding until #8289 merges to unblock CI.

/hold

File: frontend/packages/knative-plugin/src/components/knatify/KnatifyForm.tsx [hunk 1]
```diff
@@ -25,14 +24,16 @@ const KnatifyForm: React.FC<FormikProps<FormikValues> & DeployImageFormProps> =
...
-    <Form className="co-deploy-image" data-test-id="knatify-form" onSubmit={handleSubmit}>
-      <ImageSearchSection disabled />
-      <IconSection />
-      <AppSection
-        project={values.project}
-        noProjectsAvailable={projects.loaded && _.isEmpty(projects.data)}
-      />
-      <AdvancedSection values={values} />
+    <FlexForm className="co-deploy-image" data-test-id="knatify-form" onSubmit={handleSubmit}>
+      <FormBody>
```
Comment (sahil143) [inline frontend/packages/knative-plugin/src/components/knatify/KnatifyForm.tsx:28]: should use flex layout without FlexForm
```suggestion
      <FormBody flexLayout>
```

**Pull request merged**